apm-agent-core/src/test/java/co/elastic/apm/agent/metrics/builtin/SystemMetricsTest.java

```diff
@@ -32,10 +32,11 @@
     private SystemMetrics systemMetrics = new SystemMetrics();
 
     @Test
-    void testSystemMetrics() {
+    void testSystemMetrics() throws InterruptedException {
         systemMetrics.bindTo(metricRegistry);
         // makes sure system.cpu.total.norm.pct does not return NaN
         consumeCpu();
+        Thread.sleep(1000);
```

Comment:
Did the test fail otherwise?

Reply:
yes, but sporadically, depending on how I ran the tests, but never failed since I changed it